testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -436,3 +437,38 @@ def test_pathlib():
     # we really only care that pathlib
     # object handling worked
     assert u.atoms.n_atoms == 3341
+
+
+@pytest.fixture
+def large_dcdfile(tmpdir):
+    # creates a >2Gb DCD file
+    fsize = 3.8  # mb
+    nreps_reqs = int(2100 // fsize)  # times to duplicate traj to hit 2.1Gb
+
+    newf = str(tmpdir / "jabba.dcd")
+
+    u = mda.Universe(PSF, DCD)
+
+    with mda.Writer(newf, n_atoms=len(u.atoms)) as w:
+        for _ in range(nreps_reqs):
+            for ts in u.trajectory:
+                w.write(u.atoms)
+
+    yield newf, nreps_reqs
+
+
+@pytest.mark.skipif(
+    not os.environ.get("LARGEDCD", False), reason="Skipping large file test"
```

Comment:
If the env bar is supposed to skip the test then better call it SKIPLATGEFILETESTS or something like that. In any case, update CI so that it runs somewhere.